src/main/java/cc/panada/mwc/other/BlockBase.java

```diff
@@ -30,10 +54,57 @@ public BlockBase(String name, Material soundMaterial, float hardness, float resi
         setRegistryName(name);
 
         setCreativeTab(creativeTabs);
+        setHarvestLevel(harvest, harvestLevel);
 
         MainRegistry.blocks.add(this);
     }
 
+    @Override
+    public IBlockState getStateFromMeta(int meta) {
+        EnumFacing facing = EnumFacing.byIndex(meta);
+
+        if(facing.getAxis() == EnumFacing.Axis.Y)
+            facing = EnumFacing.NORTH;
+
+        return getDefaultState().withProperty(FACING, facing);
+    }
+
+    @Override
+    public int getMetaFromState(IBlockState state) {
+        return ((EnumFacing) state.getValue(FACING)).getIndex();
+    }
+
+    @Override
+    protected BlockStateContainer createBlockState() {
+        return new BlockStateContainer(this, new IProperty[] {FACING});
+    }
+
+    @Override
+    public IBlockState getStateForPlacement(World worldIn, BlockPos pos,EnumFacing facing, float hitX, float hitY, float hitZ, int meta, EntityLivingBase placer)
+    {
+        return getDefaultState().withProperty(FACING, placer.getHorizontalFacing());
+    }
+
+    @Override
+    public boolean canPlaceBlockAt(World worldIn, BlockPos pos) {
+        return super.canPlaceBlockAt(worldIn, pos) ? this.canBlockStay(worldIn, pos) : canBlockStayBoolean;
+    }
+
+    @Override
+    public void observedNeighborChange(IBlockState state, World worldIn, BlockPos pos, Block blockIn, BlockPos fromPos) {
+        if(!this.canBlockStayBoolean) {
+
+        } else {
+            worldIn.setBlockToAir(pos);
+            InventoryHelper.spawnItemStack(worldIn, pos.getX(), pos.getY(), pos.getZ(), new ItemStack(MainRegistry.WORKBENCH));
+        }
```

Comment:
Empty `if` block, remove the `!` and the brackets and just put the code currently under else under if